### PR TITLE
Persist wallet connection

### PR DESCRIPTION
**Pull request by Dodilanne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,6 +224,39 @@ export const App = () => {
...
+
+### Automatic reconnection
+When using the `useWallet` react hook, an attempt will be made to reconnect the persisted wallet on first mount.
+
+If you are not using the `useWallet` hook, you can use the `getPersistedValue` helper function to retrieve the persisted wallet and connect it during the initialization of your app:
+```typescript
+function initApp() {
+  const persistedWalletKey = getPersistedValue("connectedWallet");
+  if (persistedWalletKey) {
+    connect("nami").then((handler) => {
```
Comment (jbergeron03) [inline README.md:239]: Did you want to connect to `persistedWalletKey` instead of "nami"?

File: src/lib/react/hooks/use-wallet.hook.ts
Comment (jbergeron03) [inline src/lib/react/hooks/use-wallet.hook.ts:1]: Should we clear the persisted value on [disconnect](https://github.com/Cardano-Forge/universal-wallet-connector/pull/2/files#diff-7a7a492d4e86f14d926cb50b20ddfe8726ef7426651ac9067f851702c9c78d3cR45) ?

File: README.md (hunk 1 above)
Comment (Dodilanne) [reply README.md:239]: Yes my error

File: src/lib/react/hooks/use-wallet.hook.ts
Comment (Dodilanne) [reply src/lib/react/hooks/use-wallet.hook.ts:1]: The persisted wallet gets removed by the `deleteConnectedWallet` function which is called by `disconnect` so it should be good:)

**Pull request merged**